### PR TITLE
fix(composer): Submodel Feature Layout & Bug Fixes

### PR DESCRIPTION
**Pull request by mumanity**
## Overview
Fixes for layout in Console, SubModel layout updates, & bug fix for SubModel interactive highlights.
* Turns on features in storybook

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/scene-composer/stories/SceneComposer.stories.tsx [hunk 1]
```diff
@@ -503,7 +503,7 @@ WaterTank.args = {
     featureConfig: {
       [COMPOSER_FEATURES.SceneHierarchyRedesign]: true, // New Scene Hierarchy Panel
       [COMPOSER_FEATURES.SceneHierarchySearch]: true, // Entity Search
-      [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: false, // MultiSelect disabled, not sure if we will support this.
+      [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: true, // MultiSelect disabled, not sure if we will support this.
```
Comment (TheEvilDev) [inline packages/scene-composer/stories/SceneComposer.stories.tsx:506]: Why is multiselect on?

File: packages/scene-composer/stories/SceneComposer.stories.tsx [hunk 2]
```diff
@@ -144,13 +144,13 @@ const knobsConfigurationDecorator = [
     args.config = {
       featureConfig: {
         [COMPOSER_FEATURES.SceneHierarchyRedesign]: true, // New Scene Hierarchy Panel
-        [COMPOSER_FEATURES.SceneHierarchySearch]: false, // Entity Search
-        [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: false, // MultiSelect disabled, not sure if we will support this.
-        [COMPOSER_FEATURES.SceneHierarchyReorder]: false, // Drag/Drop Reordering
-        [COMPOSER_FEATURES.SubModelSelection]: false,
+        [COMPOSER_FEATURES.SceneHierarchySearch]: true, // Entity Search
+        [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: true, // MultiSelect disabled, not sure if we will support this.
```
Comment (TheEvilDev) [inline packages/scene-composer/stories/SceneComposer.stories.tsx:148]: We should not be turning on multi-select.

Comment (mumanity) [reply packages/scene-composer/stories/SceneComposer.stories.tsx:148]: Should I just revert the one feature or leave the file alone entirely?

Comment (mumanity) [reply packages/scene-composer/stories/SceneComposer.stories.tsx]: It is off now.

File: packages/scene-composer/stories/SceneComposer.stories.tsx [hunk 3]
```diff
@@ -144,13 +144,13 @@ const knobsConfigurationDecorator = [
...
         [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: false, // MultiSelect disabled, not sure if we will support this.
-        [COMPOSER_FEATURES.SceneHierarchyReorder]: false, // Drag/Drop Reordering
-        [COMPOSER_FEATURES.SubModelSelection]: false,
+        [COMPOSER_FEATURES.SceneHierarchyReorder]: true, // Drag/Drop Reordering
+        [COMPOSER_FEATURES.SubModelSelection]: true,
         [COMPOSER_FEATURES.ENHANCED_EDITING]: true,
         [COMPOSER_FEATURES.CameraView]: true,
-        [COMPOSER_FEATURES.EnvironmentModel]: false,
+        [COMPOSER_FEATURES.EnvironmentModel]: true,
         [COMPOSER_FEATURES.TagResize]: true,
```
Comment (sheilaXu) [inline packages/scene-composer/stories/SceneComposer.stories.tsx:154]: I think we should only turn on the features we want to release soon. check the list in console

**Pull request closed without merging**